src/__experimental__/components/input/input-presentation.spec.js

```diff
@@ -1,9 +1,37 @@
 import React from 'react';
-import { shallow } from 'enzyme';
-import { InputPresentation } from '.';
+import { shallow, mount } from 'enzyme';
+import { InputPresentation, Input } from '.';
+import baseTheme from '../../../style/themes/base';
+import 'jest-styled-components';
+
+const deletekeys = (obj, keys) => keys.forEach(k => delete obj[k]);
+const stripKeys = (obj, keys) => {
+  return Object.keys(obj).reduce((acc, key) => {
+    const strippedObj = deletekeys(obj[key], keys);
+    return { ...acc, [key]: { ...strippedObj } };
+  }, {});
+};
+
+const isUpperCase = char => char.toUpperCase() === char;
+const toCSSCase = (str) => {
```

Comment:
Seems like these methods (`deletekeys`, `stripKeys`, `isUpperCase`, `toCSSCase`) might be useful in other specs? Possibly move to a helper file?

Reply:
And `assertStyleMatch `